### PR TITLE
jennifer/issue-267

### PR DESCRIPTION
**Pull request by jaube05**
adding hover and focus styling to site, also updating home page with better link text

### CODE REVIEW

File: _layouts/home.html [hunk 1]
```diff
@@ -33,18 +33,19 @@ <h3 class="usa-card__heading">{{ post.title }}</h3>
...
     <p>Meet the humans of CivicActions Accessibility.</p>
     <ul class="usa-card-group">
         {% for author in site.people %}
         <li class="tablet:grid-col-3 usa-card">
+            <a href="{{ site.baseurl }}{{ author.url }}">
             <div class="usa-card__container">
                 <div class="usa-card__media">
                     <div class="usa-card__img">
-                        <a href="{{ site.baseurl }}{{ author.url }}"><img src="{{ site.baseurl }}/assets/img/people/{{ author.image }}" alt="{{ author.name }}"></a>
+                        <img src="{{ site.baseurl }}/assets/img/people/{{ author.image }}" alt="{{ author.name }}">
```
Comment (dmundra) [inline _layouts/home.html:48]: @jaube05 looks like axe accessibility testing wants us to give this a better alt text.

File: pages/people.html [hunk 2]
```diff
@@ -20,10 +20,11 @@
         <ul class="usa-card-group">
             {% for author in site.people %}
             <li class="tablet:grid-col-3 usa-card">
+                <a href="{{ site.baseurl }}{{ author.url }}">
                 <div class="usa-card__container">
                     <div class="usa-card__media">
                         <div class="usa-card__img">
-                            <a href="{{ site.baseurl }}{{ author.url }}"><img src="{{ site.baseurl }}/assets/img/people/{{ author.image }}" alt="{{ author.name }}"></a>
+                            <img src="{{ site.baseurl }}/assets/img/people/{{ author.image }}" alt="{{ author.name }}">
```
Comment (dmundra) [inline pages/people.html:27]: Same accessibility error.

**Review by dmundra — CHANGES_REQUESTED** [review]
@jaube05 do you have recommendations for the profile picture alt text?

File: _layouts/home.html (hunk 1 above)
Comment (jaube05) [reply _layouts/home.html:48]: Yep just saw this error too. Will look into a better alt text for images of people.

**Review by dmundra — APPROVED** [review]
Looking good. Let's see how it works on the live site.

**Pull request merged**